src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/config/CustomPostgreSQL94Dialect.java

```diff
@@ -15,6 +15,6 @@ public class CustomPostgreSQL94Dialect extends PostgreSQL10Dialect {
     public CustomPostgreSQL94Dialect() {
         super();
         this.registerHibernateType(Types.JAVA_OBJECT, NoteResource.class.getName());
-        registerFunction("contains_text", new SQLFunctionTemplate(StandardBasicTypes.BOOLEAN, "?1 @> ?2::text[]"));
+        registerFunction("contains_text", new SQLFunctionTemplate(StandardBasicTypes.BOOLEAN, "?1 && ?2::text[]"));
```

Comment:
just question, what does this mean?

Reply:
explanation here: [here](https://www.postgresql.org/docs/current/functions-array.html)